api/host.go

```diff
@@ -32,7 +32,7 @@ type (
 	// to /host/storage - a bunch of information about the status of storage
 	// management on the host.
 	StorageGET struct {
-		Folders []modules.StorageFolderMetadata `json:"folders"`
+		FolderMetadata []modules.StorageFolderMetadata `json:"foldermetadata"`
```

Comment:
FolderMetadata doesn't sound right to me. I think it's because the plurality of folder is ambiguous. It makes it sounds like it's a single folder's metadata.